packages/nimble-components/src/banner/styles.ts

```diff
@@ -0,0 +1,214 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+    BannerFail100DarkUi,
+    Black75,
+    Fail100LightUi,
+    Information100DarkUi,
+    Information100LightUi,
+    Warning100DarkUi,
+    Warning100LightUi,
+    White
+} from '@ni/nimble-tokens/dist/styledictionary/js/tokens';
+
+import {
+    actionRgbPartialColor,
+    applicationBackgroundColor,
+    bodyFont,
+    borderHoverColor,
+    buttonLabelFontColor,
+    controlHeight,
+    controlSlimHeight,
+    fillSelectedColor,
+    iconColor,
+    iconSize,
+    linkActiveFontColor,
+    linkDisabledFontColor,
+    linkFontColor,
+    standardPadding
+} from '../theme-provider/design-tokens';
+import { Theme } from '../theme-provider/types';
+import { hexToRgbaCssColor } from '../utilities/style/colors';
+import { themeBehavior } from '../utilities/style/theme';
+
+export const styles = css`
+    ${display('flex')}
+
+    :host {
+        width: 100%;
```

Comment:
I don't think we need to set this. `display: flex` is already a block level layout which fills the width.
Native block level elements like the `p` tag don't do this so I wouldn't unless we have a specific reason

Reply:
Removed